content/faq/how-rotate-kubernetes-ssh-keys.md

```diff
@@ -0,0 +1,72 @@
+-=----
+title: "How to rotate ssh keys for Kubernetes?"
+description: "Learn how to rotate ssh keys for kubernetes."
+tags:
+- kops
+- geodesic
+- kubernetes
+- faq
+---
+
+# Question
+
+How to rotate ssh keys for Kubernetes?
```

Comment:
How do we rotate the master SSH keys on a Kubernetes cluster provisioned with `kops`?